project-document/docusaurus.config.js

```diff
@@ -11,15 +11,17 @@ const config = {
   favicon: 'img/favicon.ico',
 
   // Set the production url of your site here
-  url: 'https://your-docusaurus-test-site.com', // TODO: after deciding where the host location is, write it.
+  url: 'https://genesis-tech-tribe.github.io', // TODO: after deciding where the host location is, write it.
```

Comment:
Sorry but could I ask you to delete the TODO comment? I wrote it because the URL was not decided at that time.

Reply:
Thank you for your review! That makes sense. I fixed it.